shibata added English-version file of the difference after February 10 (#110)
fujitake/vantiq-related: vantiq-platform-operations/docs/eng/create_quay.io_account.md

## About this document

> Describes the procedure for creating a new account on Quay.io.  

Reviewer: ```suggestion
This article describes the procedure for creating a new account on Quay.io.  
```

## How to create a new Quay.io account  

>   It is possible to sing in to Quay.io with a RED HAT account.       

Reviewer: ```suggestion
  You can sign in to Quay.io with a RED HAT account.       
```
